database-plugins/src/test/java/co/cask/hydrator/plugin/db/batch/action/DBQueryActionTestRun.java

```diff
@@ -61,7 +61,10 @@ public void testAction() throws Exception {
       ImmutableMap.<String, String>builder()
         .put("connectionString", getConnectionURL())
         .put("jdbcPluginName", "hypersql")
-        .put("query", "delete from \"actionTest\" where day = '${logicalStartTime(yyyy-MM-dd,0m,UTC)}'")
+        .put("jdbcPluginType", "jdbc")
+        .put("query", "delete from \"actionTest\" where day = '1970-01-01'")
```

Comment:
It wasn't working because 1970-01-01 was being set at the beginning in the table and then this is getting the logicalStartTime, which is not 1970-01-01 I believe, so I just set the times to be the same.